common/app/common/commercial/hosted/HostedPage.scala

```diff
@@ -35,9 +35,8 @@ case class HostedCampaign(
   owner: String,
   logo: HostedLogo,
   cssClass: String,
-  mainColour: String,
-  // todo: this will come through from capi once worked out how it's going to be used
-  secondaryColour: Option[String] = None,
+  brandColour: String,
+  brightFont: Boolean,
```

Comment:
Maybe better to encapsulate these two properties into a case class, eg:
```
case class CampaignColour(hexCode: String) {
    val isBright: Boolean = ???
}
```

There's a possible approach to determining brightness here:
http://stackoverflow.com/a/4680309/593836